docs/getting-started/openproject-introduction/README.md

```diff
@@ -61,53 +61,53 @@ OpenProject supports the initial set-up and configuration of a project structure
 
 | Features                   | Documentation for                                            |
 | -------------------------- | ------------------------------------------------------------ |
-| [Create a new project](#create-a-new-project)       | Create and set up a new project in OpenProject              |
-| [Set up a project structure](#project-structure) | Create a project hierarchy to structure your work in OpenProject |
-| [Project settings](#project-settings)            | Create first ideas, tasks, rough milestones.                 |
-| [Add members](#add-members)            | Invite your team to collaborate in OpenProject.              |
+| [Create a new project](../projects/#create-a-new-project)       | Create and set up a new project in OpenProject              |
+| [Set up a project structure](../../user-guide/projects/#project-structureproject-structure) | Create a project hierarchy to structure your work in OpenProject |
```

Comment:
this anchor is duplicated did you mean ../../user-guide/projects/#project-structure ?